src/app/shared/components/filters-list/filters-list.component.html

```diff
@@ -24,7 +29,12 @@
     </mat-checkbox>
   </div>
   <div class="check-boxes" fxLayout="column" fxLayoutAlign="start">
-    <p class="filter-text">Наявність ресурсу для надання освіти для осіб з ООП та осіб з інвалідністю</p>
-    <mat-checkbox color="primary" [formControl]="WithDisabilityOptionControl"> Наявні </mat-checkbox>
+    <p class="filter-text">
+      Наявність ресурсу для надання освіти для осіб з ООП та осіб з інвалідністю
+    </p>
+    <mat-checkbox color="primary" [formControl]="WithDisabilityOptionControl">
+      Наявні
+    </mat-checkbox>
   </div>
+  <app-user-radius-set *ngIf="isMapView"></app-user-radius-set>
```

Comment:
it's appears event for not map view
![image](https://user-images.githubusercontent.com/27493959/197977870-6c8b54a9-e9c7-4259-923f-3689d07f91d2.png)

Reply:
Also, can you ask the designer if we can move the radius to the top? Currently, it is hidden, so we need to scroll to this opportunity, maybe at the top it will be more user friendly

Reply:
![image](https://user-images.githubusercontent.com/27493959/197978195-3eea3714-630a-4400-9644-9135d02c2854.png)
![image](https://user-images.githubusercontent.com/27493959/197978343-b4308c47-7570-4fe3-98e7-15f47a341aed.png)